src/controllers/graphql/resolvers/post/postResolver.js

```diff
@@ -21,14 +21,21 @@ module.exports = {
     },
     getHundredxPosts: async (_, args) => {
       return await PostService.fetchHundredxResteemedPosts();
-    }
+    },
+  },
+  Mutation: {
+    broadcastPost: async (_, { permLink, title, body, tags }, { authenticatedUserInstance } ) => {
+      return !authenticatedUserInstance
+        ? new AuthenticationError('ERROR_CREATING_OR_EDITING_POST')
```

Comment:
I left a comment on this in a previous pr, but in case you miss it, could we change these ternaries to `if` statements? Ternaries should be reserved  for simple, one line operations. If you have to break apart your ternary into multiple lines, an `if` statement should probably be used

Reply:
cool.  Yup.  Worked on these in the other PR